src/components/UserSignUp.jsx

```diff
@@ -21,22 +22,35 @@ const UserSignUp = () => {
       });
   };
   return (
-    <div className="sign-up-container">
-      <form onSubmit={handleSignUp}>
-        <h1>Create Account</h1>
-        <input
-          type="email"
-          placeholder="Enter Your Email"
-          value={email}
-          onChange={(e) => setEmail(e.target.value)}></input>
-        <input
-          type="password"
-          placeholder="Enter Your Password"
-          value={password}
-          onChange={(e) => setPassword(e.target.value)}></input>
-        <button type="submit">Sign Up</button>
-      </form>
-    </div>
+    <section className="max-w-[60rem]">
+      <Card className="sign-up-container p-6 m-3">
+        <form onSubmit={handleSignUp}>
+          <div className="flex flex-col items-center gap-4">
+            <div className="text-xl">Create Account</div>
+            <div className="flex w-full flex-nowrap gap-4">
+              <Input
+                fullWidth={false}
```

Comment:
you dont need this cuz you are using flex. it would be great to add `flex-col md:flex-row`to it's parent's div, so when the window is large, email and password inputs are in one line but when the window gets smaller, two rows.

Reply:
<img width="311" alt="image" src="https://github.com/rainawan/BruinDictionary/assets/96547645/a448e82f-a887-469a-adc0-5536fb04023a">
like this

Reply:
to make email and password textboxes layered, use 
<div className="flex w-[400px] mx-auto flex-col flex-wrap mb-6 md:mb-0 gap-4">
          <Input
            size="md"
...

so that on wide screen, containers are still vertically stacked

Reply:
my code will be flex-row in wide window, and flex-col in small window.
Raina's code is setting flex-col. Apply any styles you think it should be :)

Reply:
Chose the responsive approach.

Reply:
hi

Reply:
Hi. Tiana here. Chose the responsive approach.